cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -305,6 +305,15 @@ def metadata(self) -> cirq.GridDeviceMetadata:
         """Get metadata information for the device."""
         return self._metadata
 
+    # Some user code using SerializableDevices gets the qubit list via `device.qubits`.
+    # This is a stopgap solution to prevent user breakage with the change to GridDevice.
+    @property
+    @cirq._compat.deprecated(
+        deadline='v0.16', fix='Change `device.qubits` to `device.metadata.qubit_set`.'
+    )
+    def qubits(self) -> List[cirq.Qid]:
+        return list(self._metadata.qubit_set)
```

Comment:
should this be sorted() instead of list() to prevent unintended re-orderings? Are sets guaranteed to hold their order in python now?

Reply:
Good point, updated